deploy/cruisecontrol/capacityJBOD.json

```diff
@@ -1,7 +1,33 @@
 {
   "brokerCapacities":[
     {
-      "brokerId": "-1",
+      "brokerId": "0",
```

Comment:
What is this structure used at?

Reply:
This JSON is passed to CruiseControl during integration tests and CC needs it to compute the capacities in order to provide optimization suggestions.

I made this change because retrieving the disk usage per broker returned an error indicating that the capacity is missing. This is needed for integration tests (checking that removing disks will actually lead to 0 replicas on that disk)